module-4/backend/week_three.md

```diff
@@ -1,11 +1,60 @@
 ## Week Three - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
-TBD  
+1. At a high level, what is Node?
+
+- Node is a Javascript that is capable of being written and run in a server environment.
+
+2. What is Express? What is Express similar to in the Ruby world?
+
+- Express is a framework that allows Node written applications to be used as web applications.
```

Comment:
it allows us to define endpoints for our application in a nice, easy, organized way!